### PR TITLE
fix-sanitize_param-double-quote-error

### PR DESCRIPTION
**Pull request by dreinon**

### CODE REVIEW

File: postgrest_py/utils.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def sanitize_param(param: Any) -> str:
     param_str = str(param)
     reserved_chars = ",.:()"
     if any(char in param_str for char in reserved_chars):
-        return f"%22{param_str}%22"
+        return f"\"{param_str}\""
```
Comment (leynier) [inline postgrest_py/utils.py:16]: `f'"{param_str}"'`

**Review by leynier — COMMENTED** [review]
Maybe we should use `f'"{param_str}"'` for avoid unnecessary scape

File: postgrest_py/utils.py (hunk 1 above)
Comment (dreinon) [reply postgrest_py/utils.py:16]: Does black recommend that?

Comment (dreinon) [reply postgrest_py/utils.py:16]: alright, black doesn't care

**Review by leynier — CHANGES_REQUESTED** [review]
https://github.com/supabase-community/postgrest-py/blob/master/tests/_async/test_filter_request_builder.py#L28 this test should be updated and the sync version too

**Review by leynier — APPROVED** [review]

**Pull request merged**